internal/scan.go

```diff
@@ -35,7 +36,7 @@ func Scan(cmd *cobra.Command, args []string) {
 	// Load and report out to all configured reporters
 	reporters := []reporting.Reporter{}
 
-	if !cfg.Disable_slack {
+	if slices.Contains(cfg.Reporters, "slack") {
```

Comment:
Perfect! Can you also add a check down at line 48, to make sure `"console"` is enabled?

Reply:
fixed